cyral/data_source_cyral_sidecar_listener.go

```diff
@@ -8,8 +8,9 @@ import (
 	"github.com/google/uuid"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 
```

Comment:
nit: could you please delete this blank line so that all 3rd party imports are in the same block?

Reply:
fixed